submissions/Roophee/JS - DOM/index.html

```diff
@@ -26,44 +26,35 @@
                     <ul class="nav__list">
```

Comment:
In this task try to generate menu and welcome screen by JS too.

Reply:
Done.